### PR TITLE
add shopeepay

### PR DESCRIPTION
**Pull request by candrasaputra**
## Description

Add ShopeePay payment method.

Manual Test: 
- [x] Single shipping
- [x] Multi shipping
- [x] Callback
- [x] Show/hide payment method

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Cosmetic change (text changing or color changing)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code (if necessary), particularly in hard-to-understand areas
- [x] My changes generate no new vulnerabilities
- [ ] I have added tests that prove my fix is effective or that my feature works

## Screenshot

Screen Shot 2021-09-14 at 15 23 01
Screen Shot 2021-09-14 at 15 23 11
Screen Shot 2021-09-14 at 15 24 56

### CODE REVIEW

File: Xendit/M2Invoice/Model/Payment/SHOPEEPAY.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        if ($amount < $this->dataHelper->getShopeePayMinOrderAmount() || $amount > $this->dataHelper->getShopeePayMaxOrderAmount()) {
+            return false;
+        }
+
+        if(!$this->dataHelper->getShopeePayActive()){
+            return false;
+        }
+
+        return true;
+    }
```
Comment (rendegosling) [inline Xendit/M2Invoice/Model/Payment/SHOPEEPAY.php:53]: early returns are always good 👌

**Review by rendegosling — APPROVED** [review]
Looks good for me.
is it common to not use newline endings on JS files for PHP?

File: Xendit/M2Invoice/Helper/Data.php [hunk 2]
```diff
@@ -590,6 +599,7 @@ public function xenditPaymentMethod( $payment ){
             "ovo"               => "ovo",
             "dana"              => "dana",
             "linkaja"           => "linkaja",
+            "shopeepay"               => "shopeepay",
```
Comment (kevinalfianto) [inline Xendit/M2Invoice/Helper/Data.php:602]: this is not same space as order?

Comment (candrasaputra) [reply Xendit/M2Invoice/Helper/Data.php]: updated

**Comment by candrasaputra** [comment]
@rendegosling yes

**Review by IreneGohtami — APPROVED** [review]

**Pull request merged**